csrc/nv_internal/tensorrt_llm/kernels/cutlass_kernels/moe_gemm/moe_gemm_kernels_bf16_fp8.cu

```diff
@@ -10,10 +10,10 @@
  * its affiliates is strictly prohibited.
  */
 
-#include "tensorrt_llm/kernels/internal_cutlass_kernels/src/moe_gemm/moe_gemm_template_dispatch.h"
+#include "tensorrt_llm/kernels/cutlass_kernels/moe_gemm/moe_gemm_template_dispatch.h"
```

Comment:
For these generated kernels, can we use jinja to generate them instead of explicitly materialize each of the instance?